### ISSUE TITLE
[source-orb] Missing records from Customer Table

### ISSUE THREAD

**Issue by AnilEMaharjan** [airbytehq/airbyte#34414]
### Connector Name

source-orb

### Connector Version

1.1.0

### What step the error happened?

During the sync

### Relevant information

# TLDR

It appears that the Customer stream within the Orb connector is not capturing all of the records. We realized this after we found that there were customer ids in the event that table were not in the Customer table.

We checked directly against the Orb API and these records are provided via the API, so we think this is an Airbyte connector issue

## Background

The primary event table associated with Orb is `credit_ledger_entries` and it joins to `customer` by way of `credit_ledger_entries.customer_id = customer.id`. 

A few days ago, we noticed that there were records in the `credit_ledger_entries.customer_id` that were not in the parent table and we have not been able to determine why.

We took one of the example customer ids and confirmed both within the Orb UI and the Orb API that the record exists. The only place I was not able to find it was in the records output by the connector.

This currently impacts approximately 0.75% of our records in the customers table. We believe this could date back as far as August 18, 2023. This is the first time we saw an event record that did not have an associated customer.

### Contribute

- [ ] Yes, I want to contribute
